### DISCUSSION TITLE
How to package partials for custom social identity provider

### DISCUSSION THREAD

**Discussion by kesavkolla** [keycloak/keycloak#10345] [Archive]
I am developing a custom social identity provider. I have implemented java code now I am stuck with how to package the resources. 

my provider name is customauth. I can see this provider name in the dropdown while adding identity provider. As soon as I select customauth it goes to **#/notfound**

I have the following structure under src/main/resources:

```
├── META-INF
│   └── services
│       └── org.keycloak.broker.social.SocialIdentityProviderFactory
└── theme-resources
    └── base
        └── admin
            ├── messages
            │   ├── admin-messages_en.properties
            │   └── messages_en.properties
            └── resources
                └── partials
                    ├── realm-identity-provider-customauth-ext.html
                    └── realm-identity-provider-customauth.html

```
Basically it's not picking up the theme-resources I packaged in my jar.

I am using keycloak 17.0.0 with quarkus runtime. I packaged all my provider files as a jar and kept under kc_home/provider folder. I ran build command.

Appreciate some help.

**Comment by kesavkolla** [comment]
Figured out how to package the resources. Closing this.

**Reply by inglor** [reply]
@kesavkolla want to share on how to package this?

**Reply by kesavkolla** [reply]
Followed this repo:

https://github.com/wadahiro/keycloak-discord
